packages/grid/x-data-grid/src/components/panel/filterPanel/GridFilterForm.tsx

```diff
@@ -454,7 +457,7 @@ const GridFilterForm = React.forwardRef<HTMLDivElement, GridFilterFormProps>(
               <OptionComponent key={operator.value} value={operator.value}>
                 {operator.label ||
                   apiRef.current.getLocaleText(
-                    `filterOperator${capitalize(operator.value)}` as GridTranslationKeys,
+                    `filterOperator${capitalize(operator.value)}` as 'filterOperatorContains',
```

Comment:
What was wrong with `GridTranslationKeys`?

Reply:
It could have been any value of the translation object
Some are strings, some are functions
So it complained that it could be a function

I don't know why it did not complain before

Reply:
Got it, I agree it's not worth to complicate things for this assertion 👍